src/styles/globalStyles.scss

```diff
@@ -1,27 +1,25 @@
+$crimson: #985E5B;
+$white: #FFFFFF;
+$black: #000000;
+$inputGray: #BBBDBF;
+$menuGray: #F0F0F0;
+$lightGray: #F7F7F7;
+$gray: #EDEFF2;
```

Comment:
Making a new movie are we ? Is it going to be a follow up to https://en.wikipedia.org/wiki/Fifty_Shades_of_Grey_(film)
Or will that be a sequel?